staging/src/k8s.io/kubelet/pkg/apis/podresources/v1alpha1/api.proto

```diff
@@ -39,10 +39,22 @@ message PodResources {
 message ContainerResources {
```

Comment:
Split this file in its own commit and the pb.go in its own too :)

Reply:
sure thing